lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -1869,8 +1870,11 @@ namespace {
       }
 
       switch (contextIsolation) {
-      case ActorIsolation::ActorInstance:
-      case ActorIsolation::DistributedActorInstance: {
+      case ActorIsolation::DistributedActorInstance:
+        markNearestCallAsImplicitly(/*setAsync*/None, /*setThrows*/false,
+                                    /*setDistributedThunk*/true);
```

Comment:
I don't understand this. It doesn't seem like `setThrows` can ever be `false` while `setDistributedThunk` is `true`, because you can't do a distributed call without the potential for an error to be thrown.

Reply:
Yeah I think we can collapse the two - an implicit throws (currently) always implies that it's a distributed call. I wasn't sure if we want to keep it separate, technically there could be other reasons for an implicit throw in the future I guess...?

We also need to cause the think even if the implicit throws is not in effect because the func is throwing anyway